source/tests/consistent/fitting/test_property.py

```diff
@@ -62,8 +62,9 @@
     (True, False),  # mixed_types
     (0, 1),  # numb_fparam
     (0, 1),  # numb_aparam
-    (1, 3),  # task_dim
     (True, False),  # intensive
+    ("foo", ["foo"]),  # property_name
+    (3, [4]),  # property_dim
 )
```

Comment:
It looks to me that the matrix is too large - there are 2**8=128 elements

Reply:
I delete `property_dim` key.  And fix `property_name`.